hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendStoreSystemInfo.java

```diff
@@ -62,7 +66,21 @@ public void init() {
 
     private Map<String, Object> info() {
         SchemaTransaction schema = this.graph.schemaTransaction();
-        PropertyKey pkey = schema.getPropertyKey(PK_BACKEND_INFO);
+        PropertyKey pkey = null;
+        try {
+            pkey = schema.getPropertyKey(PK_BACKEND_INFO);
+        } catch (IllegalStateException e) {
+            String message = String.format(
+                             "Should not exist schema with same name '%s'",
+                             PK_BACKEND_INFO);
+            if (message.equals(e.getMessage())) {
+                throw new HugeException("There exists multiple backend info " +
+                                        "of graph '%s' in backend '%s'",
+                                        this.graph.name(),
+                                        this.graph.backend());
+            }
+            // ignore
```

Comment:
expect throw e here, since we assume the backend is inited before calling info()